docs/DeveloperGuide.md

```diff
@@ -257,14 +257,17 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* work as teaching assistants
+* has a need to keep track of students' grades, attendance, and work submission status of relevant modules
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
-
+**Value proposition**: 
+- Fast management of students' grades over the typical GUI-driven app.
+- Easy navigation and batch processing with the help of filter and search functionality.
+- CSV file generation of student data.
```

Comment:
Should bullet points be * instead?

Reply:
Resolved by https://github.com/AY2223S1-CS2103T-T12-1/tp/pull/43/commits/7bf4eabce0a90a98255d040d7787420f35391459.